resources/translations/en.edn

```diff
@@ -37,6 +37,7 @@
             :member-email "Email"
             :member-name "Name"
             :name-and-email-required "Name and email are required."
+            :promote-to-applicant "Make applicant"
```

Comment:
I don't like this mismatch here either i.e. promote vs. make or change. We should decide on one name and be consistent, hopefully up to the event level?

Reply:
I thought promote to applicant is a clear way of expressing the command for devlopers / API users / technical folk, but users are fine with a slightly more fuzzy "Make applicant", especially since they only see the button in context.

Reply:
Sounds good, and matches the idea of having only one applicant after.

Reply:
Actually on second thought it is ambivalent